donut/constants.py

```diff
@@ -15,3 +15,9 @@
 # Length of time before recovery key expires, in minutes.
 PWD_RESET_KEY_EXPIRATION = 1 * 24 * 60
 CREATE_ACCOUNT_KEY_LENGTH = 32
+"""Value of members.gender if member's gender is unknown"""
```

Comment:
Use an enum
https://docs.python.org/3/library/enum.html
Apparently you can have multiple aliases to the same value, and you could have a custom constructor
to put all the conversion logic in one place.